aws-lc-rs/benches/data/kem_benchmarks.txt

```diff
@@ -0,0 +1,4 @@
+algorithm = KYBER512_R3
```

Comment:
Yeah, I like what they do there. The difference with KEM though is that `HKDFConfig` only has one field: the algorithm. Since `KemConfig` has fields for the keys and ciphertext it makes it more difficult to define at will. That's why right now the elements are being read from the text file.

Additionally, in the KEM benchmarking when we add new algorithms we won't have to define additional benchmarking functions. We'll simply add the algorithm to the text file and it will be automatically added to the benchmarking.

Reply:
This is also how agreement benchmarks, rsa benchmarks, etc work.

Reply:
The benchmarks for the AES algorithms also use text files and are abstracted across different benchmarking functions. But then they have the tradeoff of having to use different text files for different algorithms. I see what you're saying with the easier extendability with your current structure.

Reply:
Not really a priority, but we should make the framework in AES/cipher more consistent with what you have here